pootle/apps/pootle_store/unit/search.py

```diff
@@ -58,6 +54,14 @@ def unit_filter(self):
         return self.kwargs.get("filter")
 
     @property
+    def offset(self):
+        return self.kwargs.get("offset", None)
```

Comment:
i dont really agree - mostly because you have no way then of telling whether no offset was set or an offset of 0 was set (not that it matters in the current implementation)